src/services/allocation.js

```
@@ -1,8 +1,52 @@
+import { parseFilters } from "../util";
 import client from "./api_client";
 
+// Mock data for testing drilldown functionality when backend is not available
+const getMockData = (aggregate, filters) => {
```

can you wrap this in something that requires the user to specifically enable this? could confuse folks that misconfigured

---

Done! I've wrapped the mock data usage so it only works when explicitly enabled via the REACT_APP_USE_MOCK_DATA=true environment variable.
Changes made:
Added a USE_MOCK_DATA flag check before using mock data
Mock data is only used if the environment variable is explicitly set to true
Without this flag, real errors will be thrown instead of returning mock data when the backend is unavailable
This prevents confusion for users who misconfigured their backend - they'll see the actual connection errors instead of mock data.
To use mock data during development, you need to create a .env file with:
REACT_APP_USE_MOCK_DATA=true